LondonBritishLibrary/orient/BLorient8819.xml

```diff
@@ -0,0 +1,279 @@
+                                <locus from="70r"/> 
+                                <title ref="LIT6193HomiliaryPassions#HomilySaturday">Homily for Sunday</title>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">93</citedRange>
+                                    </bibl>, this part is the homily for Sunday, but in <ref type="mss" corresp="BNFet116"/> this homily is attributed to 
+                                    Saturday.</note>
+                                <incipit xml:lang="gez">ንንግርኬ፡ ኅዳጠ፡ ዜናሁ፡ ለፀልቦ፡ መድኃኒነ፡ መስቀል፡ ዘከመ፡ ሀሎ፡ ምስለ፡ እግዚአብሔር፡ እምቅድመ፡ ኵሉ፡ ፍጡር፤ ወእምቅድመ፡
+                                    ኵሉ፡ ግቡር፤ ትእምርተ፡ መስቀል፡ ክቡር፤ እምቅድመ፡ ነገር፤ ትእምርተ፡ መስቀል፡ አርአያሁ፡ ለእግዚአብሔር። </incipit>  
+                            </msItem>
+                            <msItem xml:id="ms_i1.9">
+                                <locus from="75r"/> 
+                                <title ref="LIT6193HomiliaryPassions#HomilySunday">Final chapter</title>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">93</citedRange>
+                                    </bibl>, this part is designated the 'final chapter', but in <ref type="mss" corresp="BNFet116"/> this homily is 
+                                    attributed to Sunday.</note>
+                                <incipit xml:lang="gez">ንግባዕኬ፡ ከመ፡ ንስብሖ፡ ለእግዚአብሔር፡ ወንወድሶ፡ እንዘ፡ ንብል፡ ጥዑመ፡ ስም፡ መድኀኒነ፡ ለሊሁ፡ ንጉሥ፡ ወእሙ፡ መቅደስ፤ 
+                                    ወመስቀሉ፡ አትሮንስ። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.10">
+                                <locus from="77v"/> 
+                                <title>Final benediction</title>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">93</citedRange>
+                                </bibl>, this part is designated the 'final benediction', but in <ref type="work" corresp="LIT6193HomiliaryPassions"/> 
+                                    this part is included to the homily attributed to Sunday.</note>
+                                <incipit xml:lang="gez">ዘጸሐፎ፡ ወለዘአጽሐፎ፡ ለዝንቱ፡ ድርሳን፡ እግዚአብሔር፡ ይጽሐፍ፡ ስሞሙ፡ በዓምደ፡ ብርሃን፡ ወያርፍቆሙ፡ በደብረ፡ ጽዮን፡ መካን፤
+                                    በሥላሴሁ፡ አመ፡ ይመጽእ፡ መድኅ<pb n="78r"/>ን፡ ምስለ፡ መላእክቲሁ፡ ትጉሃን፤ ለዓለመ፡ ዓለም። ለዘጸሐፋ፡ ለዛቲ፡ ተውዳስ፤ ወትረ፡ የሃሉ፡ በሞገስ፤ በሥጋ፡
+                                    ወነፍስ፤ እንበለ፡ ንዴት፡ ወተጽናፍስ፤ ወዘእንበለ፡ ሐተታ፡ ወሙቃስ፤፡ ኅዙነ፡ ለቡና፡ ወለተ፡ ጊጋር፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን፡ ለይኩን፤ ለይኩን። </incipit>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="78v" to="83r"/> 
+                            <title ref="LIT7017Crucifixion"/>
+                            <incipit xml:lang="gez">
+                                ሰላም፡ ለሥቃይከ፡ ለለ፩፩፤
```

Comment:
See above, on encoding rhymed text.

Reply:
Same as above. I cannot highlight the rhyme with `<l n=1></l>`. Only `<lb></lb>` would be possible, what is not verified from the catalogue description.

Reply:
https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-incipit.html for the list of elements allowed under `incipit `by TEI. It is part of `msDesc `and **not meant** for text edition, which should be provided in the `body`.
`<lb/>` is always possible, please note that it is **not** the same as `<lb></lb>` as there is never text in-between.

Reply:
(if for some reason absolutely necessary one can use `seg `around `l`, also inside `incipit`, but I think it overcomplicates the matter. note `lb `should **not** be used for poetry lines but for typographic line breaks only)